libs/pipingapp/src/lib/ui-garden/Item.tsx

```diff
@@ -46,45 +50,42 @@ const PipetestGardenItem = (props: CustomItemView<Pipetest>) => {
             hoverTimeout && clearTimeout(hoverTimeout);
             setIsOpen(false);
           }}
-          backgroundColor={'green'}
+          backgroundColor={colors.backgroundColor}
           onClick={onClick}
           style={{ width: `${columnExpanded ? 100 : width}%`, maxWidth }}
           isSelected={isSelected}
         >
           <StyledItemText>{displayName.replace('@PIPETEST-', '')}</StyledItemText>
```

Comment:
Is this necessary?

Reply:
Nop! Some bits from the old solution.
Removed :)